gnosis/safe-react
# BUG: Wrong notification for confirmation tx

This PR:
- Fixes "Confirmation tx failed" notification showing when confirmation tx was successful. The reason for this is because the notification was undefined previously.

To test:
1. Create safe with 2 owners and 2 threshold.
2. Create a transaction
3. Approve with second owner (no execution). You should see notifications: "Please sign the tx", "Confirmation tx pending", "Confirmation tx was successful"
